build.sbt

```diff
@@ -265,15 +265,10 @@ lazy val bench = project
   .settings(moduleName := "cats-bench")
   .settings(commonJvmSettings)
   .settings(
-    libraryDependencies ++= {
-      if (scalaVersion.value.startsWith("2.12"))
-        Seq(
-          "org.scalaz" %% "scalaz-core" % "7.3.6",
-          "org.spire-math" %% "chain" % "0.3.0",
-          "co.fs2" %% "fs2-core" % "0.10.7"
-        )
-      else Nil
-    },
+    libraryDependencies ++= Seq(
+      "org.scalaz" %% "scalaz-core" % "7.3.6",
+      "co.fs2" %% "fs2-core" % "3.2.8"
+    ),
```

Comment:
Thanks for all your work on this! Actually we should just drop `Chunk` from the benchmark because it creates a circular dependency with fs2. See https://github.com/typelevel/cats/pull/4194#issuecomment-1107987610.

Reply:
Hmm, I'll remove it so, thanks for noticing it.

Reply:
Removed it in https://github.com/typelevel/cats/pull/4264/commits/c3ec714f9d88b8f02bb260dfae3a4ec475293895

Now the ETA for the jmh run of `CollectiveMonoid` is just 04:18:20 for each scala version 😂

Reply:
That means that I need 13 hours of ice packs